crates/move-stackless-bytecode/src/target_filter.rs

```diff
@@ -0,0 +1,77 @@
+use move_model::model::{FunctionEnv, GlobalEnv};
+use serde::{Deserialize, Serialize};
+use std::collections::HashSet;
+
+#[derive(clap::Args, Debug, Clone, Deserialize, Serialize, Default)]
+#[clap(next_help_heading = "Filtering Options")]
+pub struct TargetFilterOptions {
+    /// Specify modules names to target
+    #[clap(long = "modules", global = true)]
+    pub modules: Option<Vec<String>>,
+
+    /// Specify functions names to target
+    #[clap(long = "functions", global = true)]
+    pub functions: Option<Vec<String>>,
+}
+
+impl TargetFilterOptions {
+    pub fn is_targeted(&self, func_env: &FunctionEnv) -> bool {
+        if let Some(modules) = &self.modules {
+            let module_name = &func_env.module_env.get_name().name().display(func_env.module_env.env.symbol_pool()).to_string();
+            if !modules.contains(&module_name) {
+                return false;
+            }
+        }
+
+        if let Some(functions) = &self.functions {
+            functions.contains(&func_env.get_name_str())
+        } else {
+            true
+        }
+    }
+
+    pub fn check_filter_correctness(&self, env: &GlobalEnv) -> Option<String> {
+        if let Some(modules) = &self.modules {
+            let mut seen = HashSet::new();
+            for module in modules {
+                if !seen.insert(module) {
+                    return Some(format!("Duplicate module `{}` found", module));
+                }
+                if env.find_module_by_name(env.symbol_pool().make(module)).is_none() {
```

Comment:
the module name is not fully qualified (that is, `<package-name>::<module-name>`). What happens if the module named `pool` exists in multiple packages in scope?

Reply:
I didn't see the answer before